pkg/gofr/datasource/opentsdb/opentsdb.go

```diff
@@ -16,6 +16,13 @@ import (
 	"go.opentelemetry.io/otel/trace"
 )
 
+// Predefined static errors.
+var (
+	ErrInvalidResponseType = errors.New("invalid response type")
```

Comment:
Just a question, why we have kept these errors exported?

Are they intended to be used by the user?